decidim-bulletin_board-ruby/lib/decidim/bulletin_board/client.rb

```diff
@@ -94,6 +99,13 @@ def publish_results(election_id)
         publish_results.call
       end
 
+      def get_status(election_id)
```

Comment:
This is the old name of the `get_election_status` method and shouldn't be included again. 😊

Reply:
It was a rebase issue :(